client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/dynamic-settings-form.js

```diff
@@ -0,0 +1,119 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import CompactCard from 'components/card/compact';
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import * as api from 'woocommerce/woocommerce-services/api';
+import { act } from 'react-dom/test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { DynamicCarrierAccountSettingsForm, CheckboxFormFieldSet } from '../dynamic-settings-form.js';
+
+configure( { adapter: new Adapter() } );
+
+jest.mock('components/forms/form-text-input', () => {
+    return function DummyFormTextField(props) {
+        return (
+        <div>
```

Comment:
(minor) looks like spacing is borked in a few places

Reply:
It looks like our eslint rule doesn't check for spacing. We should prob fix this.

Reply:
Fixed. Converted space to tabs.